### PR TITLE
Fix thread list jumping back down while scrolling

### PR DESCRIPTION
**Pull request by justjanne**
Type: Defect
Fixes: https://github.com/vector-im/element-web/issues/23727

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix thread list jumping back down while scrolling ([\#9606](https://github.com/matrix-org/matrix-react-sdk/pull/9606)). Fixes vector-im/element-web#23727. Contributed by @justjanne.

### CODE REVIEW

**Comment by justjanne** [comment]
I thought I had found an issue, but that turned out to be nothing.

File: src/components/structures/MessagePanel.tsx [hunk 1]
```diff
@@ -347,8 +347,8 @@ export default class MessagePanel extends React.Component<IProps, IState> {
         return this.eventTiles[eventId]?.ref?.current;
     }
 
-    public getTileForEventId(eventId: string): UnwrappedEventTile {
-        if (!this.eventTiles) {
+    public getTileForEventId(eventId?: string): UnwrappedEventTile | undefined {
+        if (!this.eventTiles || eventId === undefined || eventId === null) {
```
Comment (turt2live) [inline src/components/structures/MessagePanel.tsx:351]: we can simplify this to `!eventId` - an empty string as an event ID is impractical and should be impossible (if it somehow makes it this far in the stack, the calling code deserves to explode imo :p )

**Review by turt2live — APPROVED** [review]
lgtm either way

File: src/components/structures/MessagePanel.tsx (hunk 1 above)
Comment (justjanne) [reply src/components/structures/MessagePanel.tsx:351]:
```suggestion
        if (!this.eventTiles || !eventId) {
```

**Pull request merged**